krun.py

```diff
@@ -252,13 +277,17 @@ def run(self):
 
             # We dump the json after each experiment so we can monitor the
             # json file mid-run. It is overwritten each time.
-            dump_results(self.config_file, self.out_file, self.results,
-                      self.platform.audit)
+            util.dump_results(self.config_file, self.out_file, self.results,
+                              self.platform.audit)
 
             self.jobs_done += 1
             self.platform.wait_until_cpu_cool()
             self.platform.check_dmesg_for_changes()
 
+            # FIX BEFORE MERGE
```

Comment:
On Tue, Oct 27, 2015 at 04:14:23AM -0700, Sarah Mount wrote:
> >              self.jobs_done += 1
> >              self.platform.wait_until_cpu_cool()
> >              self.platform.check_dmesg_for_changes()
> >  
> > +            # FIX BEFORE MERGE
> 
> Grepping through the code, I see that this is the only ENV VAR we get. I think now we have a command line parser it would be sensible (and easier for documentation, etc.) to make this a command-line argument.

As I recall, there is also KRUN_DEBUG, which I use a lot.

I don't mind these being CLI args.

-- 
Best Regards
Edd Barrett

http://www.theunixzoo.co.uk